### ISSUE TITLE
Neos.ContentRepositoryRegistry and Neos.ContentRepository register the same Doctrine migrations

### ISSUE THREAD

**Issue by bwaidelich** [neos/neos-development-collection#4199]
The "New CR" can be installed alongside the "old" one (with https://github.com/neos/neos-development-collection/pull/4182) but since they both declare similar doctrine migrations, executing `./flow doctrine:migrate` leads to
```
PHP Fatal error:  Cannot declare class Neos\Flow\Persistence\Doctrine\Migrations\Version20110620155001, because the name is already in use in Packages/Neos/Neos.ContentRepositoryRegistry/Migrations/Mysql/Version20110620155001.php on line 10
```

**Comment by bwaidelich** [comment]
I think we can just remove all migrations from this package (see #4226)

**Comment by bwaidelich** [comment]
This has been resolved with #4240
